### DISCUSSION TITLE
Element EDGE in a scenario.

### DISCUSSION THREAD

**Discussion by Acacio98** [InterDigitalInc/AdvantEDGE#357] [Q&A] [answered]
Hey there, i am using your platform AdvantageEdge in order to deploy my own network topology to later create an app that will interact with my platform. My network topology consists on 4 zones (element type), I want to deploy 2 MEP servers, one for zone 1 and 2, and another one for zone 3 and 4 . Although there is one drawback, I can only associate one MEP server to a single zone. Would you be so kind to enlighten me on how to do it? I have noticed that in ETSI SandBox there is that topology, that is several zones share a single MEP server.

Thank you in advance for your help.

**Comment by roymx** [comment, marked as answer]
Hello @Acacio98 ,

AdvantEDGE default deployment provides a single MEC011 service instance that can be used for whole scenario (e.g. all zones)

MEC Sandbox uses a different AdvantEDGE deployment configuration where the MEC Services are deployed as scenario pods; this can be achieved with AdvantEDGE (obviously!) but requires configuration work.

We don't officially support this, so we cannot provide much support.
If you want to try...
- Comment the MEC services pods so they are not deployed as a Sandbox pods in the `.meepctl-repocfg.yaml`
```
        sandbox-pods:
        <...>
          # - meep-loc-serv
          # - meep-rnis
          # - meep-wais
          # - meep-ams
          # - meep-app-enablement
```
- Add the MEC services as Scenario pods in your scenario as per this scenario example (rename to .yaml)
[dual-mep-test.txt](https://github.com/InterDigitalInc/AdvantEDGE/files/8740295/dual-mep-test.txt)

good luck!
